src/components/ContactDetails/ContactDetailsProperty.vue

```diff
@@ -374,7 +378,35 @@ export default {
 		},
 	},
 
+	created() {
+		this.bus.$on('focus-prop', (id) => {
+			this.onFocusProp(id)
+		})
```

Comment:
```suggestion
		this.bus.$on('focus-prop', this.onFocusProp)
```

The handler won't be unregistered otherwise. Or is this a problem with the logic (especially concerning the usage of `this`)?